nix/sources.nix

```diff
@@ -32,9 +32,23 @@ let
             if spec ? tag then "refs/tags/${spec.tag}" else
               abort "In git source '${name}': Please specify `ref`, `tag` or `branch`!";
       submodules = if spec ? submodules then spec.submodules else false;
+      nixSupportsSubmodules = builtins.compareVersions builtins.nixVersion "2.4" >= 0;
```

Comment:
nit: how about moving `nixSupportsSubmodules` to the `let` on line 37? I don't think it's used outside of it

Reply:
You're right. I applied and pushed your suggestion.